### ISSUE TITLE
Restart home bridge to finish setting up this child bridge

### ISSUE THREAD

**Issue by 0x962** [prasad-edlabadka/homebridge-tuya-ir#46]
The plugin keeps asking me to restart Homebridge, even though I have done that several times already. 

To be clear, I click on the three dots on the top right hard corner of the UI, and select "restart server". I then wait for the server to come back online, and monitor the logs till I see API calls made that return the list of IR devices I am trying to connect. Once the logs end, I go to settings for the plugin to try and pair it with HomeKit and see the following. 

<img width="801" alt="Screenshot 2022-03-28 at 6 54 25 AM" src="https://user-images.githubusercontent.com/11777671/160311246-e4b3cc28-c80c-4c33-9acc-1dce4d77a301.png">

**Comment by 0x962** [comment]
For others who come across this issue, I was able to fix this by doing two things - 
1. Adding the bridge to HomeKit using the code instead of the QR scanner (the code is printed in your logs when you restart the child bridge / home bridge)
2. Turning off "auto fetch remotes" and adding device IDs for all remotes I had configured manually. 

While the UI still asks me to restart Homebridge, the integration itself works, and I am able to control my devices via HomeKit now. 

Cheers!
